winml/test/common/taefTestMacros.h

```diff
@@ -67,28 +67,19 @@ using namespace WEX::TestExecution;
 #define WINML_EXPECT_THROW_SPECIFIC(statement, exception, condition) VERIFY_THROWS_SPECIFIC(statement, exception, condition)
 #define WINML_EXPECT_HRESULT_FAILED(hresult_expression) VERIFY_FAILED(hresult_expression)
 
+#define RUNTIME_PARAMETER_EXISTS(param) \
+  bool param_value;                     \
+  SUCCEEDED(RuntimeParameters::TryGetValue(param, param_value)) && param_value;
+
```

Comment:
I don't think this compiles with the trailing semicolon

Reply:
good catch!